slack_sdk/web/async_client.py

```diff
@@ -185,7 +185,7 @@ async def admin_conversations_invite(
             user_ids (str or list): The users to invite.
         """
         kwargs.update({"channel_id": channel_id})
-        if isinstance(user_ids, list):
+        if isinstance(user_ids, (list, Tuple)):
```

Comment:
As I mentioned here: https://github.com/slackapi/python-slack-sdk/pull/894#discussion_r541457274

`isinstance(user_ids, Sequence)` does not work here as it matches str too.